Resources/doc/entity-listeners.rst

```diff
@@ -14,12 +14,8 @@ entity manager it should be registered with. Example:
             user_listener:
                 class: \UserListener
                 tags:
-                    - { name: doctrine.orm.entity_listener }
-                    -
-                        name: doctrine.orm.entity_listener
-                        event: preUpdate
-                        entity: App\Entity\User
-                        entity_manager: custom
+                    - { name: doctrine.orm.entity_listener, entity: App\Entity\User, event: preUpdate }
```

Comment:
The example shows the minimum attributes for a tag and what other attributes are available. Your changes drop this visualization. If your goal was to reduce confusion by removing one tag, you should create a new configuration block with the minimum and maximum amount of tag attributes.

You also need to adapt the XML part for the configuration to be equal to the YAML ones.

Can you please also add the `lazy` attribute to the configuration block with the maximum amount of tag attributes?